aspnetcore/release-notes/aspnetcore-10/includes/OpenApiNetV2Prev7.md

```diff
@@ -1,6 +1,6 @@
 ### Upgrade to OpenAPI.NET v2.0.0-preview7
```

Comment:
@mikekistler, This should be OpenAPI.NET v2.0.0-preview10 rather than 7, correct?  I will update if it is 10.

Also, should we refer to OpenAPI.NET instead by the the NuGet package name `Microsoft.OpenAPI` or just clarify by adding a link to the package by its name?

Reply:
@captainsafia has been incorporating new OpenApi.NET releases almost as quickly as they are publishing them. There was talk of moving up to preview13 for our preview2 but I don't know if that happened or not. But as you suggest, this is a minor item as it will eventually be replaced by the update to the GA 2.0.0 version.

As for how to refer to the package, I do not know the conventions here so whatever you think is best.

Reply:
Ugh, sorry @mikekistler , I somehow just wrote over your comment by mistake while trying to use a quote to reply to.
To recreate from memory:
- The OpenApi.NET version could be preview 13 but not clear yet.
- This info in this PR could be considered a minor sub item to the 3.1 section.
- Go with whatever the current convention is for either using OpenApi.NET the NuGet package name.
Since we used OpenApi.NET in preview 1 will just go with that, but add a link to the preview package as well as soon as I see it available.

@captainsafia, what version of OpenApi.NET should I refer to for Preview 2? v2.0.0-preview13?

Reply:
For preview2, we shipped on top of 2.0.0-preview7.

I'm fine with using OpenAPI.NET more consistently. It's a lot harder to confuse that with Microsoft.AspNetCore.OpenApi (our package).